frontends/p4/toP4/toP4.cpp

```diff
@@ -349,7 +348,7 @@ bool ToP4::preorder(const IR::Type_Extern* t) {
     setVecSep(";\n", ";\n");
     bool decl = isDeclaration;
     isDeclaration = true;
-    visit(t->methods);
+    preorder(&t->methods);
```

Comment:
Because we need to call the (custom) visit order set up by the VECTOR_VISIT macros.  This is another case where the custom visit order stuff really should go away, simplifying the code greatly.